oshi-core/src/main/java/oshi/hardware/HWDiskStore.java

```diff
@@ -62,7 +63,18 @@
      * Create an object with empty/default values
      */
     public HWDiskStore() {
-        this("", "", "", 0L, 0L, 0L, 0L, 0L, 0L, new HWPartition[0], 0L);
+        name = "";
+        model = "";
+        serial = "";
```

Comment:
you can probably just put this initialization above where the variable is defined.